assets/style/services/services.scss

```
@@ -5,3 +5,329 @@
 @import "../globalStyle/footer";
 
 // Author: Olga
+
+.global-container {
+  max-width: 1920px;
+  width: 100%;
+  margin: 0 auto;
+}
+.main__services {
+  font-family: $font;
+  min-height: 420px;
+  background-image: url(../../images/services/bg-machines.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  @media (max-width: 480px) {
+    min-height: 0;
+  }
+  .main__services-title {
+    font-weight: 700;
+    font-size: 58px;
+    text-align: center;
+    color: $text-light-color;
+    padding: 107px 0 31px 0;
```

Не рекомендую ставить паддинги дочернему элементу контейнера.
Тебе стоило задать
padding-top: 107px для контейнера .main__services
А для этой h-шки обычно стоит задавать именно
margin-bottom: 31px;
```suggestion
    margin-bottom: 31px;
```